gems/laa_multi_step_forms/spec/controllers/error_handling_spec.rb

```diff
@@ -48,20 +48,21 @@
   context 'any other error class' do
     let(:error_class) { Class.new(StandardError) }
 
-    before do
-      allow(Rails.application.config).to receive(:consider_all_requests_local).and_return(local)
-    end
-
-    context 'consider_all_requests_local is true' do
-      let(:local) { true }
+    context 'non production RAILS_ENV' do
+      before do
+        ENV['RAILS_ENV'] = 'development'
+      end
```

Comment:
So this is a change to global state inside the tests suite which wont be reset automatically... this is rarely a good idea.. there are two solutions:

1.
```ruby
around do |spec|
  rails_env = ENV['RAILS_ENV']
  ENV['RAILS_ENV'] = 'development'
  spec.run
  ENV['RAILS_ENV'] = rails_env
end
```
2.
```
before do
  allow(ENV.to receive(:[]).and_call_original
  allow(ENV.to receive(:[]).with('RAILS_ENV').and_return('development')
end
```
I prefer the second as it is a pure mocking solution.. :) 

> NOTE: These code snippets are untested and may not 100% as is